gflownet/envs/crystals/crystal.py

```diff
@@ -535,3 +541,242 @@ def readable2state(self, readable: str) -> List[int]:
             + self.space_group.readable2state(readables[2])
             + self.lattice_parameters.readable2state(readables[3])
         )
+
+    @torch.no_grad()
+    def top_k_metrics_and_plots(
```

Comment:
This is great, and actually great enough to not be here for crystals only but actually for all environments and proxies in general. In other words, is there anything here that is specific to the crystal environment or the crystal proxy? It seems to me that the answer is no, or if anything, it could be changed easily. Thus, I think this should go in `env/base.py` or (better) in a new `utils/plotting.py` (or the like). What do you think?

Reply:
I agree. I'll do that